lib/Conversion/TTNNToEmitC/TTNNToEmitC.cpp

```diff
@@ -666,11 +667,14 @@ void populateTTNNToEmitCPatterns(mlir::MLIRContext *ctx,
   // Eltwise binary ops
   //
   patterns.add<EltwiseBinaryOpConversionPattern<ttnn::AddOp>,
+               EltwiseBinaryOpConversionPattern<ttnn::SubtractOp>,
```

Comment:
Please revert these changes (line 670-671).

Reply:
I tried putting bitwise and logical ops next to each other in all parts of code, just to have cleaner grouping. Same here, grouping on the right seemed better than the one on the left.

<img width="2011" alt="image" src="https://github.com/user-attachments/assets/ee19d38c-3c9c-4cb1-8085-af4b137c1a14" />

If you still prefer the former way, I'll revert the changes.

Reply:
I prefer it with Kristijan's changes, the grouping makes more sense, let's leave it like that.

Reply:
Asif agreed in an online discussion.